docs/src/components/Sidenav/index.tsx

```diff
@@ -1,23 +1,40 @@
 import * as React from "react";
 import styled, { css } from "styled-components";
+import { Portal, mediaQueries } from "@kiwicom/orbit-components";
 import Menu from "@kiwicom/orbit-components/lib/icons/MenuHamburger";
 import Close from "@kiwicom/orbit-components/lib/icons/Close";
 import mq from "@kiwicom/orbit-components/lib/utils/mediaQuery";
 import useClickOutside from "@kiwicom/orbit-components/lib/hooks/useClickOutside";
 
+import { CONTENT_PADDING } from "../../consts";
+
 interface WrapperProps {
   width: number;
   shown: boolean;
 }
 
+export const HEADER_HEIGHT = "52px"; // safely above the search input height
+export const paddingMixin = css`
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
```

Comment:
could you `padding: 0.5rem 0` instead? I see wrong `padding-left` was removed below

Reply:
I don't think it's a good practice for mixins, especially responsive ones. Imagine that I set left and right padding to the element that I'm applying the mixin to, that means that the mixin would reset that padding on larger screens, which would be unexpected because the function of the padding is only to share vertical padding between sidenav and navigation (the name sucks, though). I think having robust styles is worth this small DRY sacrifice.

Reply:
in my opinion, there is no need for that padding mixin at all. Especially when you export it from `SideNav` and use it inside `Navbar`, for such kind of mixins should be then some common folder and it's used only on two places